appeals/e2e/cypress/page_objects/appealsListPage.js

```diff
@@ -55,38 +69,44 @@ export class AppealsListPage extends Page {
 	}
 
 	clickReviewAppellantCase(position) {
-		this.clickAccordionByText('Case documentation');
+		this.clickAccordionByButton('Documentation');
 		this.basePageElements
 			.tableCell()
-			.eq(position - 2)
+			.eq(position - 1)
 			.find(this.selectors.link)
 			.click();
 	}
 
 	clickChangeVisitTypeHasCaseTimetable() {
-		this.clickAccordionByText('Site details');
-		cy.get(
-			'#accordion-default1024-content-2 > .govuk-summary-list > :nth-child(7) > .govuk-summary-list__actions > .govuk-link'
-		).click();
+		this.clickAccordionByText('Timetable');
+		cy.get('.appeal-site-visit > .govuk-summary-list__actions > .govuk-link').click();
 	}
 
 	clickChangeVisitTypeHasSiteDetails() {
-		this.clickAccordionByText('Site details');
-		cy.get(':nth-child(8) > .govuk-summary-list__actions > .govuk-link').click();
+		this.clickAccordionByText('Site');
+		cy.get(
+			':nth-child(3) > .govuk-accordion__section-header > .govuk-accordion__section-heading > .govuk-accordion__section-button'
+		)
+			//cy.get('.appeal-site-visit > .govuk-summary-list__actions > .govuk-link')
+			.click();
 	}
 	fillInput(text, index = 0) {
 		this.basePageElements.input().eq(index).clear().type(text);
 	}
 
 	fillInput1(text, index = 1) {
```

Comment:
This function looks a little strange on first glance

It doesn't do anything differently over `fillInput` except have the index argument preset to 1. However this argument could be provided to `fillInput()` itself when it is called

If this function was argued to be useful to keep, even then it could just call the original `fillInput()` but pass in a different index argument.

Reply:
Agree, will discuss this with you. Or we can refine once we have the data attribute.